src/pages/lib/graphqlapi/upgrade-guide/q/platform/[platform].mdx

```diff
@@ -0,0 +1,138 @@
+export const meta = {
+  title: `Upgrade guide`,
+  description: `Learn how to upgrade from AWS AppSync SDK for JavaScript (Maintenance mode) to AWS Amplify.`
+};
+
+If you're using AWS AppSync SDK for JavaScript (Maintenance mode), we recommend you to upgrade to AWS Amplify library or use the [AWS AppSync Apollo links](https://github.com/awslabs/aws-mobile-appsync-sdk-js#aws-appsync-links-for-apollo-v3) with the Apollo GraphQL client.
+
+**What does Maintenance mean?**
+While in maintenance, the AWS AppSync SDK for JavaScript (V2) will continue to receive updates that ensure compatibility with backend services and security updates. No new features will be introduced in the AWS AppSync SDK for JavaScript (V2).
+
+**How long will Maintenance last?**
+AWS AppSync SDK for JavaScript (V2) will be in Maintenance for 12 months i.e until June 30, 2024, after which no new updates will be made. This may result in the APIs going out of sync with the services and result in breaking production apps.
+
+## Option 1: Upgrade to AWS Amplify (Recommended)
+AWS Amplify provides APIs for querying, mutating, and subscribing to data in AppSync with less configuration overhead. Here's an example of running a query in the AWS AppSync SDK for JavaScript (Maintenance mode) vs. AWS Amplify API category:
+
+**AWS AppSync SDK for JavaScript (Maintenance mode):**
+```js
+import { createtodo } from './graphql/queries'
+import awsconfig from './aws-exports'
+import AWSAppSyncClient from "aws-appsync";
+
+const client = new AWSAppSyncClient({
+  url: awsconfig.aws_appsync_graphqlEndpoint,
+  region: awsconfig.aws_appsync_region,
+  auth: {
+    type: awsconfig.aws_appsync_authenticationType,
+    apiKey: awsconfig.aws_appsync_apiKey,
+  },
+});
+
+const result = await client.mutate({
+  mutation: gql(createTodo),
+  variables: {
+    input: {
+      name: 'New Todo',
+      description: 'Description of my new todo',
+    }
+  }
+})
+```
+
+**AWS Amplify:**
+```js
```

Comment:
Since we have TS + JS examples everywhere else in the docs, can we add that here, as well?

Reply:
This is more for illustration purposes. Customers can deep dive in the respective Amplify sections.